requirements.txt

```diff
@@ -14,6 +14,9 @@ xlrd==1.0.0
 ipython
 gitpython
 pygraphviz
+sphinx_rtd_theme
+Sphinx
+numpy==1.15.4
```

Comment:
What for ? Is it linked to OAuth2 ?

Reply:
Pas du tout!

Reply:
Alright, can you remove it ?

Reply:
numpy is required by pandas for data analysis

Reply:
Pandas should work without numpy.
Also, [`mezzanine-organization` don't seems to use numpy](https://github.com/Ircam-Web/mezzanine-organization/search?q=numpy).

For `Sphinx` and `sphinx_rtd_theme`, it seems that you removed it a few weeks ago because they were obsolete : https://github.com/Ircam-Web/mezzanine-organization/commit/d73c0c1fa52fdd2f29e355868c8b8022479274f8#diff-4d7c51b1efe9043e44439a949dfd92e5827321b34082903477fd04876edb7552

Reply:
numpy is installed by pandas

Reply:
`pandas` installs `numpy` automatically. No need to add it in requirements.txt.
If you need a specific version of `numpy` (because the version automatically installed by `pandas` fails to build), please add a comment with a link to the issue.